catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -2319,9 +2319,12 @@ protected ResourceResponse getResource(ResourceRequest resourceRequest, boolean
 
         } catch (RuntimeException e) {
             LOGGER.error("RuntimeException caused by: ", e);
+            if (StringUtils.contains(e.getMessage(), "data usage limit exceeded")) {
```

Comment:
Testing the exception message for a certain string doesn't seem like the best solution. Is there a way we could avoid doing this?